doc/nrf/ug_thread_commissioning.rst

```diff
@@ -22,24 +22,24 @@ Commissioning roles
+* *Leader* - Watches over the network configuration, allows Commissioner Candidate to become Commissioner, and ensures there is only one active Commissioner.
+* *Commissioner Candidate* - Role for a device capable of becoming the Commissioner, but has not yet been assigned the role of Commissioner.
+* *Commissioner* - Authenticates the Joiner and provides it with network credentials required to join the network.
   The Commissioner can be part of the network or it can be outside the network (see :ref:`thread_ot_commissioning_types`).
-* Joiner - Role for a new device that wants to join the Thread network.
+* *Joiner* - Role for a new device that wants to join the Thread network.
   It exchanges messages with the Commissioner through the directly connected Joiner Router that serves as a proxy.
-* Joiner Router - Role for a router device that is one hop away from the Joiner device in the Thread network and is the sole device connected with the Joiner.
+* *Joiner Router* - Role for a router device that is one hop away from the Joiner device in the Thread network and is the sole device connected with the Joiner.
   Responds to the Discovery Request of the Joiner.
   Moreover, when chosen by the Joiner, it passes subsequent communication in a secure manner.
-* Border Router - Role for a device equipped with at least two interfaces (for example Thread and Wi-Fi, Ethernet, LTE etc.) that forwards data between a Thread network and a non-Thread network.
+* *Border Router* - Role for a device equipped with at least two interfaces (for example Thread and Wi-Fi, Ethernet, LTE etc.) that forwards data between a Thread network and a non-Thread network.
   It can also be an interface for the Commissioner.
 
-  * Border Agent function, usually combined with the Border Router role, accepts :ref:`petitions <thread_ot_commissioning_phases>` from the Commissioner candidate.
-    It also relays commissioning messages between Thread Network and a Commissioner.
-    If a Commissioner is a part of the network, the same node is both the Commissioner and the Border Agent.
+  The Border Router role is usually combined with the *Border Agent* function, which accepts :ref:`petitions <thread_ot_commissioning_phases>` from the Commissioner candidate.
+  It also relays commissioning messages between Thread Network and a Commissioner.
+  If a Commissioner is a part of the network, the same node is both the Commissioner and the Border Agent.
 
   .. note::
-        While the Border Router is typically used only in the external commissioning process, the Nordic Border Router, which is based on the OpenThread Border Router (OTBR), supports both on-mesh and external commissioning.
+     While the Border Router is typically used only in the external commissioning process, the Nordic Border Router, which is based on the OpenThread Border Router (OTBR), supports both on-mesh and external commissioning.
```

Comment:
I'm not sure if there are plans to maintain/support the current Nordic Border Router.
@MarekPorwisz, should we remove this note?

Reply:
There are no plans to support Nordic border router baser on raspberry pi. OTBR should be used. Please remove this note